frontend/src/store/types.ts

```diff
@@ -40,4 +63,32 @@ export type ModalDialogState =
   | { kind: "delete-methodology"; methodology: MethodologyDto }
   | { kind: "methodology-description"; methodology: MethodologyDto }
   | { kind: "create-edit-methodology"; editing?: MethodologyDto }
-  | { kind: "create-edit-project"; editing?: ProjectDto };
+  | { kind: "create-edit-project"; editing?: ProjectDto }
+  | { kind: "failed-recompute-ic50"; reason: string };
+
+export type ExperimentsTableSize = "expanded" | "minimized";
+
+export function negateTableSize(size: ExperimentsTableSize): ExperimentsTableSize {
+  if (size === "expanded") return "minimized";
+  return "expanded";
+}
+
+export const chartColors = [
+  "#C6E294",
+  "#8E95D5",
+  "#E6D85D",
+  "#3D9C97",
+  "#C15959",
+  "#53AC62",
+  "#A26BC4",
+  "#6076E0",
+  "#374275",
+  "#BF5688",
+];
```

Comment:
Don't you want to use color generator?

Reply:
no, there was a relevant discussion on the slack where we agreed on predefined set of color from figma